app_flutter/test/utils/output.dart

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) 2019 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+
+import 'package:flutter/foundation.dart';
+import 'package:flutter_test/flutter_test.dart';
+
+typedef AsyncVoidCallback = Future<void> Function();
+
+Future<void> checkOutput({
```

Comment:
nit: Add a doc comment explaining  this checks blocks use of debugPrint

Reply:
added docs.